testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/x509/AbstractX509AuthenticationTest.java

```diff
@@ -100,6 +102,27 @@ protected boolean isImportAfterEachMethod() {
         return true;
     }
 
+    @BeforeClass
+    public static void onBeforeTestClass() {
+        if (System.getProperty("auth.server.container", "undefined").endsWith("wildfly")) {
```

Comment:
This won't work wit auth-server-eap.

Reply:
True, forgot about eap ... I guess it should be enough to include a check of "eap" there ?

Reply:
I believe something like that would work too:
```java
Boolean.parseBoolean(System.getProperty("auth.server.jboss"));
```